templates/web/base/report/display_tools_mobile.html

```diff
@@ -0,0 +1,43 @@
+<div class="shadow-wrap">
+    <ul id="key-tools-mobile" class="js-key-tools">
+      [% IF c.user_exists OR NOT problem.non_public %]
+        [% IF c.cobrand.users_can_hide AND relevant_staff_user %]
+        <li><form method="post" action="/report/[% problem.id %]/delete" id="remove-from-site-form">
+            <input type="hidden" name="token" value="[% csrf_token %]">
+            <button type="submit" id="key-tool-report-abuse" class="has-inline-svg has-inline-svg--column-wrap" data-confirm="[% loc('Are you sure?') %]" name="remove_from_site">
+              <svg role="presentation" aria-hidden="true" xmlns="http://www.w3.org/2000/svg" width="1.5em" height="1.5em" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0z"/><path fill="currentColor" d="M12.866 3l9.526 16.5a1 1 0 0 1-.866 1.5H2.474a1 1 0 0 1-.866-1.5L11.134 3a1 1 0 0 1 1.732 0zM11 16v2h2v-2h-2zm0-7v5h2V9h-2z"/></svg>
```

Comment:
This template could use the same icons/ templates I think? The contents of the SVGs look the same.

Reply:
(Then, if all that's different then is the order of the text and the SVG, can flex order fix that? Then they could share more of the same template for the list items themselves.)